core/module.h

```diff
@@ -441,6 +443,7 @@ class alignas(64) Module {
 
   std::vector<bess::IGate *> igates_;
   std::vector<bess::OGate *> ogates_;
+  uint64_t deadends_;
```

Comment:
Shouldn't you have a counter per worker as multiple workers may be running through the module at the same time. As such deadends() would return the sum of the counters for each worker.

Reply:
good catch! adjusted